### PR TITLE
Add and Improve Exchanges, Merchandise and Roadmap

### PR DESCRIPTION
**Pull request by jhepkema**
Added NEO/BTC and NEO/ETH on KuCoin

### CODE REVIEW

**Comment by jhepkema** [comment]
@revett I'm not sure if we should add merchandise, but I think people will be looking for a list of merch, we might as well add it in the awesome-neo repo. I made a file in /resources to prevent cluttering the readme.md. Your thoughts?

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 <h1 align="center">Awesome NEO</h1>
 
 <p align="center">
-  A curated list of awesome <b><a href="https://neo.org/">NEO</a></b> libraries, applications and resources.
+  A curated list of awesome <b><a href="https://neo.org/">NEO</a></b> libraries, applications and resources.<br/>
+  If you want to contribute to this list, please follow the <a href="https://github.com/CityOfZion/awesome-neo/blob/master/CONTRIBUTING.md">contribution guidelines</a>.
```
Comment (revett) [inline README.md]: Nice idea adding this - can it go at the bottom of the README please.

File: README.md [hunk 2]
```diff
@@ -90,18 +92,30 @@
...
 - [CoinSpot](https://www.coinspot.com.au/buy/ans) - An Australian exchange that allows trading for Australian Dollars (AUD).
 - [CoinSwitch](https://www.coinswitch.co/exchange/btc/neo) - An Indian exchange that allows trading for various other cryptocurrencies.
+- [CryptoMate](https://cryptomate.co.uk/buy-neo/) - A British exchange that allows purchases using GBP.
 - [Cryptopia.co](https://www.cryptopia.co.nz/Exchange?market=NEO_BTC) - A New Zealand-based exchange that allows trading for BTC, LTC and DOGE.
 - [HitBTC](https://hitbtc.com/NEO-to-ETH) - A European exchange that allows trading for BTC, ETH and USDT.
 - [JUBI](https://www.jubi.com/coin/ans) - A Chinese exchange that allows trading for Chinese Yuan Renminbi (CNY). *Chinese interface*
-- [Poloniex](https://www.19800.com/trade.html?marketid=cny_gas) - A United States-based exchange that allows trading for BTC and ETH (only GAS).
+- [KuCoin](https://www.kucoin.com/#/trade/NEO-BTC) - A Korean exchange that allows trading for BTC and ETH. *English, Chinese, Japanese, Dutch, Russian, Korean and Portuguese interfaces*
```
Comment (revett) [inline README.md]: A-Z please.

File: resources/neo-merchandise.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  >
+</p>
+
+<h1 align="center">NEO Merchandise</h1>
+
+<p align="center">
+  A curated list of merchandise for Neo.
+</p>
+
+## Contents
```
Comment (revett) [inline resources/neo-merchandise.md]: A-Z order of these please.

File: README.md (hunk 1 above)
Comment (jhepkema) [reply README.md]: Sure thing

File: README.md (hunk 2 above)
Comment (jhepkema) [reply README.md]: Do you mean you want GAS above NEO? I believe the exchanges in each section are in fact in A-Z order; Poloniex does not trade NEO so I removed it from the NEO list.

**Comment by jhepkema** [comment]
Should be good now! @revett

**Comment by revett** [comment]
@jhepkema A few PRs have been merged ahead of you, so you now have conflicting changes. 

Could you bring your branch up to date, and then I'll be happy to review again 👍

**Comment by jhepkema** [comment]
@revett should be good now

**Comment by jhepkema** [comment]
Oh I now see it is still conflicting, I might redo the entire PR to fix it once I'm back home

**Pull request closed without merging**